### ISSUE TITLE
[Idea] Material variation options

### ISSUE THREAD

**Issue by MauveCloud** [MauveCloud/Ic2ExpReactorPlanner#69]
From #33:

> 9.add an option for using GT recipes since it is different on some components
39.divide advanced alloy further down into iron, tin, and bronze(copper and tin) when not using GT recipes

For the GT recipes, I will only apply them when there's a single set of materials for the alternate recipe.  I also plan to add an option for whether the coolant cell is made by filling an empty cell (like in 1.7.10) or a universal fluid cell (like in later MC versions - not sure exactly when the switch happened, but IC2 v2.8 for MC 1.12.2 has no "empty cell")

@KrisBigK for the advanced alloy option, it's not entirely clear whether you want a sub-option for showing "bronze" as a fundamental material, or just automatically add the copper and tin content of the bronze to the materials list when the option is selected.

**Comment by KrisBigK** [comment]
For the coolant cell, I think it is better to just let the user choose the MC version and the GT version (if using GT) that they use. IIRC GT5 have a million recipes for advanced alloy, and you might not want to get into that. 
For bronze, I would go with only an option for showing bronze as a fundamental one, because 1) other mods might have different recipes on bronze and 2) it is fairly easy to calculate the needed tin and copper afterwards.

**Comment by MauveCloud** [comment]
> For the coolant cell, I think it is better to just let the user choose the MC version and the GT version (if using GT) that they use. IIRC GT5 have a million recipes for advanced alloy, and you might not want to get into that.

That's actually a good idea.  I've been trying to keep the planner "version agnostic", but that runs into problems like A. reflectors have different durability in 1.7.10 than in 1.12.2, and B. iridium neutron reflectors are GT-only in 1.7.10, but available with pure IC2 in 1.12.2.

> For bronze, I would go with only an option for showing bronze as a fundamental one, because 1) other mods might have different recipes on bronze and 2) it is fairly easy to calculate the needed tin and copper afterwards.

The only alternate recipes for bronze I can confirm via the FTB wiki are the GT inefficient dust-crafting (to encourage building an alloy smelter) and tetrahedrite plus tin in an alloy smelter, neither of which are likely to be used by a player who is ready to make a nuclear reactor.  I have a vague memory of a mod that did 1 copper and 1 tin to make 2 bronze, but I can't remember what mod that was.

**Comment by KrisBigK** [comment]
GT5u has builds for 1.10.2 (latest: 5.10.41) although it is now obsolete. Also GTCE is for 1.12.2 (I can't remember exactly if it has reactor components)

**Comment by MauveCloud** [comment]
iirc, the developer(s) dropped support for the GT5u 1.10.2 builds before they really got past alpha, so I don't consider it worth supporting in the planner.  GTCE is separated from IC2 to the point that GTCE cables do not connect to IC2 machines, the crops are gone (the textures are left behind in the assets folder of the jar last I checked, but still with mixed-case names that won't work in 1.12.2), and the GT reactor components are gone (technically there's a "neutron reflector" from GTCE, but it won't go into an IC2 nuclear reactor, and it doesn't seem to have any use in GTCE at the moment)

> Since you added the version selection, I don't think that "Enable GT 5.08/09 Components" is necessary because only when people are using GT do they need to use those components.

Agreed.  I meant to remove those, not sure how I failed to notice them still being there.  I'll remove them in the next version.
